src/wallet/PegOut.cpp

```diff
@@ -19,7 +20,7 @@ mw::Transaction::CPtr PegOut::CreatePegOutTx(
     std::vector<Input> inputs = WalletUtil::SignInputs(input_coins);
 
     // Calculate fee
-    const uint64_t fee = WalletUtil::CalculateFee(fee_base, input_coins.size(), 1, 2);
+    const uint64_t fee = fee_base * Weight::Calculate({ .num_kernels = 1, .num_owner_sigs = 1, .num_outputs = 2 });
```

Comment:
The fee needs to be added to amount before calling SelectCoins.

Reply:
Good catch

Reply:
I've fixed this, but I'm going to go through and remove fee_base from the SelectCoins call in a future PR.